health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/LatLongMap/LatLongMap.js

```
@@ -0,0 +1,412 @@
+  const tooltip = (dataTip) => {
+    return (
+      dataTip.name &&  <div
+        style={{
+          display: "flex",
+          padding: "5px",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
+      >
+        {<div style={{ fontWeight: 700, fontSize: "16px", margin: "10px" }}>{getTitleHeading(dataTip.name)}</div>}
+        {chartData.hasOwnProperty(dataTip.name.toLowerCase()) && dataTip.value ? (
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)